roles/common/templates/etc/pip.conf

```diff
@@ -0,0 +1,2 @@
+[global]
+index-url = {{ common.pip.proxy_url|default('') }}
```

Comment:
I think you'd want to do something more like:
{% if common.pip.proxy_url is defined %}
[global]
index-url = {{ common.pip.proxy_url }}
{% endif %}